app/src/main/java/com/kms/cura/view/TypeSelection.java

```diff
@@ -0,0 +1,39 @@
+package com.kms.cura.view;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageButton;
+import android.widget.LinearLayout;
+
+import com.kms.cura.R;
+
+public class TypeSelection extends AppCompatActivity implements View.OnClickListener {
```

Comment:
This class needs better naming. First if it's inherited from an Activity class then it's name should end with "Activity". Also the "Type" part is ambiguous. Should be more specific about which type that this activity is for.